### DISCUSSION TITLE
Is nuxt 3 mature enough for production in this time?

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#17987] [Questions]
Hi, I am working on Vue for several years. I want to move my Vue project to Nuxt but I see Nuxt 3 is already released. Nuxt 2 is available but as I convert my frontend I want to build by the latest version. Is nuxt 3 mature enough for production? If not, when it will release the final public version?

**Comment by wobsoriano** [comment]
https://github.com/nuxt/framework/discussions/2876

https://github.com/nuxt/framework/discussions/2282

https://github.com/nuxt/framework/discussions/2198

https://github.com/nuxt/framework/discussions/2198#discussioncomment-1951587
